docs/canonicalk8s/capi/explanation/capi-ck8s.md

```diff
@@ -19,31 +21,47 @@ allowing for a variety of Kubernetes distributions to be delivered in all of
 the supported infrastructure providers. {{product}} is one such Kubernetes
 distribution that seamlessly integrates with Cluster API.
 
+## {{product}} and CAPI
+
 With {{product}} CAPI you can:
 
-- provision a cluster with:
-    - Kubernetes version 1.31 onward
-    - risk level of the track you want to follow (stable, candidate, beta, edge)
-    - deploy behind proxies
-- upgrade clusters with no downtime:
-    - rolling upgrades for HA clusters and worker nodes
-    - in-place upgrades for non-HA control planes and worker nodes
+- Provision a cluster that meets your needs
+  - Choose Kubernetes version 1.31 onwards
+  - Choose the risk level of the track (Kubernetes version) you want to follow -
+  stable, candidate, beta or edge
+  - Deploy behind proxies
+- Upgrade clusters with no downtime
+    - Carry out rolling upgrades for High Availability (HA) clusters and worker
+    nodes
+    - Carry out in-place upgrades for non-HA control planes and worker nodes
 
-Please refer to the [tutorial section] for concrete examples on CAPI
-deployments.
+Please refer to the [getting started tutorial] for a concrete example of a CAPI
+deployment.
 
 ## CAPI architecture
 
 Being a cloud-native framework, CAPI implements all its components as
-controllers that run within a Kubernetes cluster. There is a separate
-controller, called a ‘provider’, for each supported infrastructure substrate.
-The infrastructure providers are responsible for provisioning physical or
-virtual nodes and setting up networking elements such as load balancers and
-virtual networks. In a similar way, each Kubernetes distribution that
-integrates with ClusterAPI is managed by two providers: the control plane
-provider and the bootstrap provider. The bootstrap provider is responsible for
-delivering and managing Kubernetes on the nodes, while the control plane
-provider handles the control plane’s specific lifecycle.
+controllers that run within a Kubernetes cluster. There is a separate controller
+, called a ‘provider’, for each supported
+infrastructure substrate.
```

Comment:
```suggestion
controllers that run within a Kubernetes cluster.
```

This feels out of place, it feels like it belongs in `### Infrastructure provider` section.

Reply:
Thank you. I missed that when I was reformatting.